charts/newrelic-infrastructure-beta/templates/_helpers.tpl

```diff
@@ -0,0 +1,234 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "newrelic.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "newrelic.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+
+{{/* Generate mode label */}}
+{{- define "newrelic.mode" }}
+{{- if .Values.privileged -}}
+privileged
+{{- else -}}
+unprivileged
+{{- end }}
+{{- end -}}
+
+{{/* Common labels */}}
+{{- define "newrelic.labels" -}}
+helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+app.kubernetes.io/name: {{ include "newrelic.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+mode: {{ template "newrelic.mode" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
```

Comment:
I would suggest to run `helm create` on a different directory and see which labels are the new standard generated by helm. Then we can incorporate those here :)